custom/icds_reports/management/commands/custom_data_pull_ap.py

```diff
@@ -0,0 +1,88 @@
+import csv
+from datetime import date
+
+from dateutil.relativedelta import relativedelta
+from django.core.management.base import BaseCommand
+from django.db import connections, transaction
+
+from corehq.sql_db.connections import get_icds_ucr_citus_db_alias
+
+query_to_fetch_awcs = """
```

Comment:
you could use orm here. Class name is AWCLocation

Reply:
updated